sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Listeners/BlobListener.cs

```diff
@@ -4,25 +4,38 @@
 using System;
 using System.Threading;
 using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
 using Microsoft.Azure.WebJobs.Extensions.Storage.Common.Listeners;
 using Microsoft.Azure.WebJobs.Host.Listeners;
 using Microsoft.Azure.WebJobs.Host.Scale;
+using Microsoft.Extensions.Logging;
 
 namespace Microsoft.Azure.WebJobs.Extensions.Storage.Blobs.Listeners
 {
     internal sealed class BlobListener : IListener, IScaleMonitorProvider
     {
         private readonly ISharedListener _sharedListener;
+        private readonly ILogger<BlobListener> _logger;
 
         private bool _started;
         private bool _disposed;
+        private string _details;
 
-        public BlobListener(ISharedListener sharedListener)
+        // for mock test purposes only
+        internal BlobListener(ISharedListener sharedListener)
         {
             _sharedListener = sharedListener;
         }
 
-        public Task StartAsync(CancellationToken cancellationToken)
+        public BlobListener(ISharedListener sharedListener, BlobContainerClient containerClient, IBlobPathSource blobPathSource, ILoggerFactory loggerFactory)
+        {
+            _sharedListener = sharedListener;
+            _details = $"blob container={containerClient.Name}, storage account name={containerClient.AccountName}, blob name pattern={blobPathSource.BlobNamePattern}";
```

Comment:
Let's use "blob container name" to be consistent with "queue name" and "storage account name".

Reply:
Done